### PR TITLE
Replacing IO and IA with actual Internal Tribunal Agent

### PR DESCRIPTION
**Pull request by projectkepler-ru**
## About the Pull Request
What this  do is cut out the Investigation officer and Investigation Agent from EZ and make them their own role

idea of this was courtesy of @wolf751 

the IO and IA are not guards to be ordered by guard commander, never has been.

They are under the supervision of the ECL and ITDO.

## Why It's Good For The Game

1. The separation of IO and IA from EZ was long overdue, especially with the pushes for more roleplaying standard, this establish one of the important step toward the server taking itself more seriously.
2. The IO and IA were always intended as a counter-balance to LCZ mistreatment of D-Class, HCZ causing problem with science or EZ being generally unpleasent. 
3. Most importantly, they are given the full authority to roam the site as necessary but they were often disallowed from entering departmental area due to nobody actually understanding what the IO/IA Title entails and only see the hud icon
4. Having to get admin to send ISD Every time something happens is, generally a huge waste of time for admin as they need to get 2 confirmation before they can send it. That and we cannot send multiple response team at the same time due to players limitation, there is, simply not enough dead people to send 2+ MTF team at the same time.
5. There is already a higher expectation than usual for any role that directly deal with security, this is the same. Except, perhaps even higher with this role.
6. A certain undocumented change has touched IO/IA Equipment without me being notified, While I am able to check the gitblame and has already figured out who did it, I would much rather that this is avoided entirely. 

## Proof of Testing
We want to confirm the PR Work
image
Blue

image
image
image
image

## Changelog

:cl:
add: The Internal Tribunal Agent && The Internal Tribunal Operative alt title, These are intended to work with ECL and ITDO as necessary to maintain the site, up to 6 of them may spawn at any given time.
add: A new area dedicated to the Tribunal Agent so that they can perform all the necessary lab function
qol: The Forensic Briefcase now hold security tape and  camera film roll, alongside actually having them to begin with
/:cl:

### CODE REVIEW

**Comment by checkraisefold** [comment]
new vending machine useless but this is interesting
I feel this has overlap with the tribunal officer role since they can *already* do investigations themselves. however we already had investigation officer before, and I feel a dedicated security person with forensics skills is good for this. this will give tribunal officer a bit more meaning and more actually getting reports and investigations so I like it

**Comment by checkraisefold** [comment]
HOWEVER I DONT WANT 6 OF THEM. DO NOT SPAWN 6. SPAWN A MAX OF 1 OR *MAYBE* 2

**Comment by projectkepler-ru** [comment]
> HOWEVER I DONT WANT 6 OF THEM. DO NOT SPAWN 6. SPAWN A MAX OF 1 OR _MAYBE_ 2

There are  up to 6 investigation agent in the round prior to this PR, this is meant to approximate how many you can get

They're cut down to 3, at most.
We want 1 for both liason and 1 on stand by at all time

**Comment by projectkepler-ru** [comment]
> new vending machine useless but this is interesting I feel this has overlap with the tribunal officer role since they can _already_ do investigations themselves. however we already had investigation officer before, and I feel a dedicated security person with forensics skills is good for this. this will give tribunal officer a bit more meaning and more actually getting reports and investigations so I like it

Irrelevant to description of this PR
1. This separation was made in direct response to modification being made to the IO and IA role as it was a necessary motivation to actually separate the role, it was made a year ago and I found myself the 5 hours of freetime needed to test it
2.  the purpose of this role is to fufil what was written for it already, to be an auditor for security, the role existed preceding the existence of internal tribunal officer but it was never updated to account for that. That being said, I've no intention of changing the tribunal officer in any notable way. However, the title is very, very rarely used for it's intended purpose because it was really hard to explain what it does without messaging everyone what it does, because you never gave me the wiki account I asked for and never added it to the wiki
3. New Vendor is indeed a bloat that could be cut down later, for now it's there as an intended item vendor.

**Comment by checkraisefold** [comment]
> There are up to 6 investigation agent in the round prior to this PR, this is meant to approximate how many you can get
> They're cut down to 3, at most. We want 1 for both liason and 1 on stand by at all time

one person usually ever picked investigation agent. there were not 6 investigation agents used all the time there were 6 ez agent slots with an alt title of investigation agent. it does not apply here, you've overhauled it into a cl4 ZC command department tag job that should *not* have 6 slots. it should have a max of 2, not 3, and even that's pushing it given its access levels.

> We want 1 for both liason and 1 on stand by at all times

what?? why?? you want 1 for the ECL, 1 for the ITDO, and one just as an extra bonus guy to do random stuff?
this is a 'Tribunal Department Agent'. key word 'tribunal department' meaning intended for the tribunal department, the ECL can investigate ethical violations himself since that's his entire job and maybe delegate to the tribunal department agent/security if its something requiring forensic or other evidence/arrests to be done

> hard to explain what it does

okay, explain what tribunal officer does that the brand new wiki page at https://scp13.miraheze.org/wiki/Internal_Tribunal_Department_Officer does not

> because you never gave me the wiki account I asked for and never added it to the wiki

you never DMed me asking for this. I don't remember you asking for this. you sent one message mentioning me in admin-chat which I didn't see and made no concerted effort to actually get one. but that's completely irrelevant to this PR
image

**Comment by checkraisefold** [comment]
continuing on;

> Having to get admin to send ISD Every time something happens is, generally a huge waste of time for admin as they need to get 2 confirmation before they can send it

what 2 confirmation? i don't need any confirmation from anyone to send ISD as a gamemaster
you don't need to send 2 teams either at the same time, the 1 ISD investigator on-site as staff can handle everything going on. also ISD is uuusuually sent only in big cases that even on site tribunal wouldnt be able to handle since we issue job bans

> A certain undocumented change has touched IO/IA Equipment without me being notified, While I am able to check the gitblame and has already figured out who did it, I would much rather that this is avoided entirely.

welcome to the nature of free and open source software, sponsored by your code will be touched and modified. this will always happen no matter what you do. and I looked at the git blame, an alt title outfit should not randomly have much higher access than the normal title, that is stupid as hell and its actually good that it got changed

**Comment by projectkepler-ru** [comment]
> one person usually ever picked investigation agent. there were not 6 investigation agents used all the time there were 6 ez agent slots with an alt title of investigation agent. it does not apply here, you've overhauled it into a cl4 ZC command department tag job that should _not_ have 6 slots. it should have a max of 2, not 3, and even that's pushing it given its access levels.

this is a rough approximation of how many slot, it'll have 3 for now

> > We want 1 for both liason and 1 on stand by at all times
> 
> what?? why?? you want 1 for the ECL, 1 for the ITDO, and one just as an extra bonus guy to do random stuff? this is a 'Tribunal Department Agent'. key word 'tribunal department' meaning intended for the tribunal department, the ECL can investigate ethical violations himself since that's his entire job and maybe delegate to the tribunal department agent/security if its something requiring forensic or other evidence/arrests to be done

As written

This was always the intended purpose of the said auditor before the internal tribunal was ever even coded and has existed preceding that. 1 on stand by of all time does not mean them doing random stuff, it mean that you need one who will be  on the office for further instructions 


> okay, explain what tribunal officer does that the brand new wiki page at https://scp13.miraheze.org/wiki/Internal_Tribunal_Department_Officer does not

The IA and IO as originally written served the purpose of being directly involved with the site itself, not offering advices but acting directly to reprimand personnel when any violation is reported. It was created preceding even the usage of ISD on the server, that, is why this is being made at all. The purpose of the role existed already, it being made separate is necessary

Due to the way it was ever made, it was extremely unobvious to outsider on what does the role do unless they were explicitly written to them, You, failed to understand that quotation and shortened the message to fuckall, thanks.



> > because you never gave me the wiki account I asked for and never added it to the wiki
> 
> you never DMed me asking for this. I don't remember you asking for this. you sent one message mentioning me in admin-chat which I didn't see and made no concerted effort to actually get one. but that's completely irrelevant to this PR image

you still did not

**Comment by projectkepler-ru** [comment]
> what 2 confirmation? i don't need any confirmation from anyone to send ISD as a gamemaster you don't need to send 2 teams either at the same time, the 1 ISD investigator on-site as staff can handle everything going on. also ISD is uuusuually sent only in big cases that even on site tribunal wouldnt be able to handle since we issue job bans

You'd need to, 1st, be actually informed via fax and notified something is happening as it is not always that staff will witness an incident happening

2nd, be then confirmed by witnessing it or by looking  into it  before they send it


> welcome to the nature of free and open source software, sponsored by your code will be touched and modified. this will always happen no matter what you do. and I looked at the git blame, an alt title outfit should not randomly have much higher access than the normal title, that is stupid as hell and its actually good that it got changed

In the case of SS13
a bad PR Body && Changelog warrants blocking until it is resolved.

**Comment by oprimeolt** [comment]
I'm not sure if you ever saw it but EZsec can usually deal with that stuff no need to add another role to just add to the chaos as EC and ITD can just talk to the role that exists to resolve issues with security and punish them the GC or if there are none the EZ ZC or at wprst tell the SD to send EZsec to deal with them, no need for a new role that exists to basically antagonise sec players

File: code/game/jobs/job/command.dm [hunk 1]
```diff
@@ -245,6 +245,67 @@
 	mechanical_difficulty = "Easy"
 	duties = "Ensure that proper ethics is upheld, both with the treatment of personnel and of SCPs. Communicate with the Ethics Committee. Keep security in line."
 
+// IO  && IA Moved into a singular role
+/datum/job/invesigation_agent
```
Comment (checkraisefold) [inline code/game/jobs/job/command.dm]: `invesigation_agent` -> `tribunal_agent`

File: code/game/jobs/job/command.dm [hunk 2]
```diff
@@ -245,6 +245,67 @@
...
+	title = "Internal Tribunal Department Agent"
+	department = "Command"
+	selection_color = "#2f2f7f"
+	department_flag = COM|SEC
+	total_positions = 3 // A rough estimate of how many IA you can get if you compacted them
+	spawn_positions = 3
+	supervisors = "Internal Tribunal Department Officer & Ethic Committee Liason"
```
Comment (checkraisefold) [inline code/game/jobs/job/command.dm]: `Ethic Committee Liason` -> `Ethics Committee Liaison`

File: code/game/jobs/job/command.dm [hunk 3]
```diff
@@ -245,6 +245,67 @@
...
+	economic_power = 4
+	requirements = list(EXP_TYPE_COMMAND = 720, EXP_TYPE_SECURITY = 900, EXP_TYPE_BUR = 60)
+	minimal_player_age = 10
+	ideal_character_age = 27
+	alt_titles = list("Internal Tribunal Departmen Operative" = /decl/hierarchy/outfit/job/security/investigation_operative)
```
Comment (checkraisefold) [inline code/game/jobs/job/command.dm]: `Departmen` -> `Department`

File: code/datums/outfits/jobs/security.dm [hunk 4]
```diff
@@ -325,6 +293,41 @@
 	head = /obj/item/clothing/head/helmet/scp/hczsecurityguard
 	gloves = /obj/item/clothing/gloves/tactical/scp
 
+// IA
+
+/decl/hierarchy/outfit/job/security/investigation_agent
```
Comment (checkraisefold) [inline code/datums/outfits/jobs/security.dm]: `investigation_agent` -> `tribunal_agent`

File: code/datums/outfits/jobs/security.dm [hunk 5]
```diff
@@ -325,6 +293,41 @@
...
+	belt = /obj/item/storage/belt/holster/security/fullrhino
+	l_hand = /obj/item/storage/briefcase/crimekit
+	glasses = /obj/item/clothing/glasses/sunglasses/sechud
+	head = /obj/item/clothing/head/det/grey
+	gloves = /obj/item/clothing/gloves/forensic
+	l_pocket = /obj/item/handcuffs
+	r_pocket = /obj/item/melee/telebaton
+	backpack_contents = list(/obj/item/clothing/head/helmet/scp/hczsecurityofficer = 1, /obj/item/clothing/suit/armor/pcarrier/scp/medium = 1, /obj/item/crowbar/emergency_forcing_tool = 1, /obj/item/gun/energy/stunrevolver = 1)
+
+/decl/hierarchy/outfit/job/security/investigation_operative
```
Comment (checkraisefold) [inline code/datums/outfits/jobs/security.dm]: `investigation_operative` -> `tribunal_operative`

File: code/datums/outfits/jobs/security.dm [hunk 6]
```diff
@@ -325,6 +293,41 @@
...
+	suit = /obj/item/clothing/suit/armor/pcarrier/scp/tactical
+	l_pocket = /obj/item/handcuffs
+	r_pocket = /obj/item/melee/telebaton
+	backpack_contents = list(/obj/item/handcuffs = 1)
+	belt = /obj/item/storage/belt/holster/security/fullusp45
+	head = /obj/item/clothing/head/helmet/scp/hczsecurityguard
+	mask = /obj/item/clothing/mask/balaclava
+	gloves = /obj/item/clothing/gloves/tactical/scp
+
+	backpack_contents = list(/obj/item/clothing/head/helmet/scp/hczsecurityofficer = 1, /obj/item/crowbar/emergency_forcing_tool = 1, /obj/item/gun/energy/stunrevolver = 1, /obj/item/ammo_magazine/box/a45 = 1)
```
Comment (checkraisefold) [inline code/datums/outfits/jobs/security.dm]: 2 helmets one in backpack and one already on head - redundant

File: code/game/jobs/job/command.dm [hunk 7]
```diff
@@ -245,6 +245,67 @@
...
+	outfit_type = /decl/hierarchy/outfit/job/security/investigation_agent
```
Comment (checkraisefold) [inline code/game/jobs/job/command.dm]: `investigation_agent` -> `tribunal_agent`

File: code/game/jobs/job/command.dm [hunk 8]
```diff
@@ -245,6 +245,67 @@
...
+	max_skill = list(
+		SKILL_COMBAT      = SKILL_MASTER,
+	    SKILL_WEAPONS     = SKILL_MASTER,
+	    SKILL_FORENSICS   = SKILL_MASTER
+	)
+	skill_points = 25
+
+	roleplay_difficulty = "Hard - Very Hard"
+	mechanical_difficulty = "Medium - Hard"
+	duties = "Ensure Foundation protocols are followed. Keep security in line. Protect the Ethics Committee Liason and Tribunal Officer"
```
Comment (checkraisefold) [inline code/game/jobs/job/command.dm]: add a period to the end
`liason` -> `liaison`

File: code/game/jobs/job/command.dm [hunk 9]
```diff
@@ -245,6 +245,67 @@
...
+
+
```
Comment (checkraisefold) [inline code/game/jobs/job/command.dm]: fix extra whitespace (remove 1 newline)

**Review by checkraisefold — CHANGES_REQUESTED** [review]
general grammar problems

File: code/game/jobs/job/security.dm [hunk 10]
```diff
@@ -60,6 +60,61 @@
...
+	spawn_positions = 3
+	supervisors = "Internal Tribunal Department Officer & Ethic Committee Liaison"
+	economic_power = 4
+	requirements = list(EXP_TYPE_COMMAND = 720, EXP_TYPE_SECURITY = 900, EXP_TYPE_BUR = 60)
+	minimal_player_age = 10
+	ideal_character_age = 27
+	alt_titles = list("Internal Affair Operative" = /decl/hierarchy/outfit/job/security/tribunal_operative)
+	outfit_type = /decl/hierarchy/outfit/job/security/tribunal_agent
+	class = CLASS_B
+	hud_icon = "hudbridgeofficer"
```
Comment (checkraisefold) [inline code/game/jobs/job/security.dm]:
```suggestion
	hud_icon = "hudtdia"
```
your newly added hud icon isnt being used!

File: code/game/jobs/job/security.dm [hunk 11]
```diff
@@ -60,6 +60,61 @@
 	mechanical_difficulty = "Easy - Medium"
 	duties = "Manage all three Security branches. Keep track of potential and on-going threats (such as containment breaches). Work with other departments to respond to said threats."
 
+// IO  && IA Moved into a singular role
+/datum/job/tribunal_agent
+	title = "Internal Affair Agent"
```
Comment (checkraisefold) [inline code/game/jobs/job/security.dm]:
```suggestion
	title = "Internal Affairs Agent"
```

File: code/game/jobs/job/security.dm [hunk 12]
```diff
@@ -60,6 +60,61 @@
...
+	department = "Command"
+	selection_color = "#2f2f7f"
+	department_flag = COM|SEC
+	total_positions = 3 // A rough estimate of how many IA you can get if you compacted them
+	spawn_positions = 3
+	supervisors = "Internal Tribunal Department Officer & Ethic Committee Liaison"
```
Comment (checkraisefold) [inline code/game/jobs/job/security.dm:71]:
```suggestion
	supervisors = "Internal Tribunal Department Officer & Ethics Committee Liaison"
```

File: maps/site53/icons/assignment_hud.dmi
Comment (checkraisefold) [inline maps/site53/icons/assignment_hud.dmi:1]: it's the same gold color as the ITDO; maybe you can have it be a white colored gavel instead to signify lower level or you can keep it

**Review by checkraisefold — CHANGES_REQUESTED** [review]

File: code/datums/outfits/jobs/security.dm [hunk 13]
```diff
@@ -325,6 +293,40 @@
...
+	suit = /obj/item/clothing/suit/armor/pcarrier/scp/tactical
+	l_pocket = /obj/item/handcuffs
+	r_pocket = /obj/item/melee/telebaton
+	belt = /obj/item/storage/belt/holster/security/fullusp45
+	head = /obj/item/clothing/head/helmet/scp/hczsecurityguard
+	mask = /obj/item/clothing/mask/balaclava
+	gloves = /obj/item/clothing/gloves/tactical/scp
+
+	backpack_contents = list(/obj/item/crowbar/emergency_forcing_tool = 1, /obj/item/gun/energy/stunrevolver = 1, /obj/item/ammo_magazine/box/a45 = 1)
 /* Non-SCP security outfits */
```
Comment (checkraisefold) [inline code/datums/outfits/jobs/security.dm:330]:
```suggestion
	backpack_contents = list(/obj/item/crowbar/emergency_forcing_tool = 1, /obj/item/gun/energy/stunrevolver = 1, /obj/item/ammo_magazine/box/a45 = 1)

/* Non-SCP security outfits */
```
whitespace

File: code/game/objects/items/weapons/cards_ids.dm [hunk 14]
```diff
@@ -704,6 +704,16 @@ GLOBAL_LIST_EMPTY(conversion_cards)
...
+// IA - Stop fucking with the access, you know who you are
+/obj/item/card/id/zcseclvl4ia
+	name = "security ID"
+	desc = "A teal ID. Looks cool."
+	icon_state = "securitylvl4"
+	item_state = "Sec_ID4"
+	job_access_type = /datum/job/tribunal_agent
+	class = CLASS_B
+
+
```
Comment (checkraisefold) [inline code/game/objects/items/weapons/cards_ids.dm]:
```suggestion
```
extra newline

**Review by checkraisefold — CHANGES_REQUESTED** [review]
whitespace

File: code/game/jobs/job/security.dm (hunk 10 above)
Comment (projectkepler-ru) [reply code/game/jobs/job/security.dm]: I added a new icon? I didn't remember doing that

File: code/game/jobs/job/security.dm [hunk 15]
```diff
@@ -60,6 +60,61 @@
...
 	duties = "Manage all three Security branches. Keep track of potential and on-going threats (such as containment breaches). Work with other departments to respond to said threats."
 
+// IO  && IA Moved into a singular role
+/datum/job/tribunal_agent
+	title = "Internal Affairs Agent"
+	department = "Command"
+	selection_color = "#2f2f7f"
+	department_flag = COM|SEC
+	total_positions = 3 // A rough estimate of how many IA you can get if you compacted them
+	spawn_positions = 3
```
Comment (checkraisefold) [inline code/game/jobs/job/security.dm:70]:
```suggestion
	total_positions = 1
	spawn_positions = 1
```
veto'd for game balance reasons, change this and the rest of the PR is fine

File: code/game/machinery/weapon_vending.dm [hunk 16]
```diff
@@ -134,3 +134,22 @@
...
+		/obj/item/ammo_magazine/box/a57/ap = 6,
+		/obj/item/ammo_magazine/box/a57/hp = 6,
+		/obj/item/ammo_magazine/box/a45 = 8,
+		/obj/item/ammo_magazine/scp/usp45 = 12,
+		/obj/item/ammo_magazine/box/a357 = 6,
+		/obj/item/ammo_magazine/scp/p90_mag/ap = 24,
+		/obj/item/ammo_magazine/scp/p90_mag/hp = 24,
+		/obj/item/gun/energy/stunrevolver = 6,
+	)
+
```
Comment (checkraisefold) [inline code/game/machinery/weapon_vending.dm:155]: i just looked at the actual product list and this is a bit crazy. please just have them spawn with a reasonable gun loadout and ahve them go to ez armory if they need advanced firepower for whatever reason (which they shouldnt) or to restock on ammo

**Review by checkraisefold — CHANGES_REQUESTED** [review]
game balance changes

**Comment by projectkepler-ru** [comment]
I'm completely resigning on this part, "Veto'd" is completely rediculous.

**Pull request closed without merging**